content/blog/move-manifestjson-to-outdirs-parent-directory-in-vite-5/index.md

```diff
@@ -0,0 +1,154 @@
+---
+{
+  title: "Move manifest.json to the Output Directory's Parent in Vite 5",
+  description: "Move Vite's manifest.json file outside of the build directory.",
+  published: '2024-02-07',
```

Comment:
Just checking - did we want to post this to the 7th? It might be harder to find on the UU site as a result given that we have other articles newer than this 😊

Up to you!

Reply:
Let's keep it!

I thought it looked weird with two Vite posts right next to each other in the articles list so I back-dated it to when I published it 😅

Reply:
Makes sense to me!